configuration/autostart.md

```diff
@@ -197,7 +197,7 @@ sudo systemctl disable magicmirror.service
 Create file `/home/server/.config/lxsession/LXDE-pi/autostart` with the
 following contents:
 
-```
+```sh
```

Comment:
is there a rule for when to use shel and when to use sh?

Reply:
No official rule. For scripts I use `sh` and otherwise `shell` or `bash`. Should I use `shell` everywhere?

Reply:
i see. so all other occurences in the docs are following this too then?

Reply:
I admit that I have not checked this in detail. I only noticed code blocks here without any indication of how they are to be rendered. 

I have just checked, so far only `shell` has been used.

Actually, the distinction as I have just explained it makes little sense, for the user it is displayed the same anyway. I just harmonised it.